src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -2557,6 +2589,34 @@ class Simplex_tree {
   /** \brief Upper bound on the dimension of the simplicial complex.*/
   int dimension_;
   bool dimension_to_be_lowered_ = false;
+
+  // MULTIPERS STUFF
+ public:
+  /**
+   * \brief Sets the number of parameters of the filtrations if SimplexTreeOptions::is_multi_parameter. 
+   * */
+  void set_number_of_parameters(int num) { 
```

Comment:
Is it intended that we can change the number of parameters of an existing Simplex_tree, or should we ask for it in the constructor?

Reply:
It can be useful to change the filtration of an existing simplextree, e.g., adding a filtration while preserving the old ones. But I'm already asking it at construction time in the python interface.